### PR TITLE
Fixes #13075 - Added padding between host#show status icon and caption

### PR DESCRIPTION
**Pull request by alongoldboim**

### CODE REVIEW

File: app/assets/stylesheets/patternfly_and_overrides.scss [hunk 1]
```diff
@@ -284,3 +284,11 @@ div.form-group .glyphicon-info-sign{
 .col-md-3 b{
   font-size: 100%;
 }
+
+span.status-error{
```
Comment (dLobatog) [inline app/assets/stylesheets/patternfly_and_overrides.scss]: This works well, but it's missing status-warn and status-question. Mind to group them all under the same css rule like:

``` css
span.status-error, span.status-ok, span.status-warn, span.status-question {
  padding-left: 3px;
}
```

**Comment by alongoldboim** [comment]
@dLobatog fixed.

**Comment by dLobatog** [comment]
Merging, looks like the integration test doesn't have anything to do with this PR

**Comment by dLobatog** [comment]
Merged as fa2a0c76d093fcfcf313097fd3eb5098ad0c3e7b, thanks @alongoldboim!

**Pull request closed without merging**